Misc/NEWS.d/next/Library/2019-09-06-17-40-34.bpo-37953.db5FQq.rst

```diff
@@ -0,0 +1 @@
+Forward references in the typing module now have improved __hash__ and __eq__ methods.

```

Comment:
Could we slightly modify this news entry to be more specific to the class the methods are being added to? Also fixes the formatting to use codeblocks for `__hash__` and `__eq__` as well as adding a Sphinx role to `ForwardReferences` and `typing`. This allows readers to click on them to be linked to the corresponding documentation. 

```suggestion
In :mod:`typing`, improved the ``__hash__`` and ``__eq__`` methods for
:class:`ForwardReferences`.
```

Note that this change also requires this to be split into a second line, which I've included in the suggestion. This is because it goes above the 80 character width limit for reST.